### PR TITLE
Pr/floating label

### PR DESCRIPTION
**Pull request by niklaushug**
Current implementation requires a polyfill for css pseudo selector `:focus-within` like https://github.com/jonathantneal/focus-within. Microsoft browser does not support it yet.

`:focus-within` makes code much cleaner and lighter than using a react library like https://github.com/ferdaber/react-focus-within.

Do we already other polyfills included in tocco-client? What do you think about polyfills in general?

### CODE REVIEW

File: packages/tocco-ui/src/StatedValue/StatedValue.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  } else if (dirty) {
+    return stylingCondition.WARNING
+  }
+}
+
+/**
+ * Wrap <EditableValue> and <FormattedVaule> in <StatedValue> to describe it
+ * and signal conditions.
+ */
+const StatedValue = props => {
```
Comment (MethodenMann) [inline packages/tocco-ui/src/StatedValue/StatedValue.js]: I dont know if i should like the name "StatedValue". Maybe we can discuss it. But isnt it just a new made "FormField"?

File: packages/tocco-ui/src/StatedValue/StyledStatedValue.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+import {
+  declareFocus,
+  declareFont,
+  fontScale,
+  shadeColor,
+  spaceScale
+} from '../utilStyles'
+
+/*
+TODO
```
Comment (MethodenMann) [inline packages/tocco-ui/src/StatedValue/StyledStatedValue.js]: i also prefer the polifilly over the react componnent. But shouldn't we load the polyfill with webpack.

Comment (MethodenMann) [reply packages/tocco-ui/src/StatedValue/StyledStatedValue.js]: https://github.com/jonathantneal/focus-within
it this a "real" polyfill? Its important is only overwrites browser behavior if the the browser not supports the feature.

File: packages/tocco-ui/src/StatedValue/StatedValue.js (hunk 1 above)
Comment (niklaushug) [reply packages/tocco-ui/src/StatedValue/StatedValue.js]: FormFieldFloat

File: packages/tocco-ui/src/StatedValue/StatedValue.js [hunk 3]
```diff
@@ -0,0 +1,106 @@
+import PropTypes from 'prop-types'
```
[the remark this answers is not in the document]
Comment (niklaushug) [reply packages/tocco-ui/src/StatedValue/StatedValue.js]: see 70ae957362a4d0539ab616c90d55eefeb7a55c94

File: packages/tocco-ui/src/StatedValue/StatedValue.js [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+      }
+    </StyledStatedValueWrapper>
+  )
+}
+
+StatedValue.propTypes = {
+  /**
+   * A component to enter or display data.
+   */
+  content: PropTypes.node,
```
[the remark this answers is not in the document]
Comment (niklaushug) [reply packages/tocco-ui/src/StatedValue/StatedValue.js]: see 70ae957362a4d0539ab616c90d55eefeb7a55c94

File: packages/tocco-ui/src/StatedValue/StyledStatedValue.js (hunk 2 above)
Comment (niklaushug) [reply packages/tocco-ui/src/StatedValue/StyledStatedValue.js]: see 2e35cdf0404838189dd0b804937b6ede184ed8e4

**Pull request merged**